### DISCUSSION TITLE
Emby: update failed

### DISCUSSION THREAD

**Discussion by LiveDrunk** [tteck/Proxmox#2847] [Help Needed]
Hello,
I wanted to update my Emby server using the script but i receive an error:
 ✓ Stopped Emby
 \ Updating Emby   
[ERROR] in line 65: exit code 0: while executing command dpkg -i emby-server-deb_${LATEST}_amd64.deb &> /dev/null

Nothing left then to reboot and Emby comes up again but not updated of course.

I'm on Version [4.8.3.0]
Emby Server 4.8.5.0 is Available

Am i missing something? 
I'm not sure where i can adjust the ${LATEST} parameter so it is found.

**Comment by tteck** [comment]
What does this command show?
```
curl -sL https://api.github.com/repos/MediaBrowser/Emby.Releases/releases/latest | grep '"tag_name":' | cut -d'"' -f4
```
I show 4.8.5.0

**Reply by LiveDrunk** [reply]
I get 4.8.5.0 as well
![image](https://github.com/tteck/Proxmox/assets/35400092/3058d2f6-2d79-4fa2-babd-c9aa6cb668c7)
![image](https://github.com/tteck/Proxmox/assets/35400092/b60993aa-202a-43b1-b939-735a2ffcd5fb)

update command returns:
![image](https://github.com/tteck/Proxmox/assets/35400092/9c47e5f5-9293-4592-bba6-a733440f7a00)

**Reply by LiveDrunk** [reply]
Sorry still no change, i can see that the latest version is downloaded but it won't move beyond that point.
Any suggestions or things i can do?
